packages/subgraph/src/packages/token/token-voting.ts

```diff
@@ -54,8 +56,8 @@ export function _handleProposalCreated(
     proposalEntity.votingMode = VOTING_MODES.get(parameters.votingMode);
     proposalEntity.supportThreshold = parameters.supportThreshold;
     proposalEntity.minParticipation = parameters.minParticipation;
-    proposalEntity.startDate = parameters.startDate;
-    proposalEntity.endDate = parameters.endDate;
+    //proposalEntity.startDate = parameters.startDate;
+    //proposalEntity.endDate = parameters.endDate;
```

Comment:
remove if not necessary

Reply:
5cc5807247354835b830df5694ff2a7f33392a44